contracts/Colony.sol

```diff
@@ -48,7 +48,9 @@ contract Colony is ColonyStorage {
     // Initialise the task update reviewers
     IColony(this).setFunctionReviewers(0xda4db249, 0, 2); // setTaskBrief => manager, worker
     IColony(this).setFunctionReviewers(0xcae960fe, 0, 2); // setTaskDueDate => manager, worker
-    IColony(this).setFunctionReviewers(0xbe2320af, 0, 2); // setTaskPayout => manager, worker
+    IColony(this).setFunctionReviewers(0x6aaf4a77, 0, 2); // setTaskManagerPayout => manager, worker
```

Comment:
It doesn't make sense to get the worker to agree to what the manager payout should be. Just don't permission the `setTaskManagerPayout` function for now.